frontend/src/ui/Blocks/Series.tsx

```diff
@@ -125,25 +125,49 @@ const ReadySeriesBlock: React.FC<ReadyProps> = ({
     const finalTitle = title ?? (showTitle ? series.title : undefined);
 
     const events = series.events.filter(event =>
-        !isPastLiveEvent(event.syncedData?.endTime ?? null, event.isLive));
+        !isPastLiveEvent(event.syncedData?.endTime ?? null, event.isLive)
+        && !isUpcomingEvent(event.syncedData?.startTime ?? null));
+
+    const upcomingEvents = series.events.filter(event =>
+        isUpcomingEvent(event.syncedData?.startTime ?? null));
 
     const sortedEvents = [...events];
-    sortedEvents.sort(match(order, {
-        "NEW_TO_OLD": () => compareNewToOld,
-        "OLD_TO_NEW": () => compareOldToNew,
-    }, unreachable));
+    sortedEvents
+        .sort(match(order, {
+            "NEW_TO_OLD": () => compareNewToOld,
+            "OLD_TO_NEW": () => compareOldToNew,
+        }, unreachable))
+        .sort((a, b) => +b.isLive - +a.isLive);
+
+    if (upcomingEvents.length === 1) {
```

Comment:
Maybe add a very short comment about this special case?

```suggestion
    // If there is only one upcoming event, we don't bother with the extra box for it.
    if (upcomingEvents.length === 1) {
```